edge-modules/load-gen/config/settings.json

```diff
@@ -1,8 +1,8 @@
 {
   "messageFrequency": "00:00:00.02",
   "twinUpdateFrequency": "00:00:05",
-  "jitterFactor": 0.5,
   "transportType": "amqp",
   "messageSizeInBytes": 1024,
-  "outputName": "output1"
+  "outputName": "output1",
+  "startDelay": 2
```

Comment:
What does it means? 2 mins or 2 seconds?  Please use TimeSpan like messageFrequency.

Reply:
+1 on timespan.. or rename it to startDelaySecs